index.js

```diff
@@ -605,6 +605,11 @@
            other.unicode === this.unicode;
   }
 
+  //  RegExp$prototype$lte :: RegExp ~> RegExp -> Boolean
+  function RegExp$prototype$lte(other) {
+    return lte(toString(this), toString(other));
+  }
```

Comment:
I think string comparison is reasonable for _exact_ match patterns like `/abc/`. I'm not sure that the idea of comparing `/\b([01]+b|\d+|[\da-f]+h)\b/` with something like `/\b\d+ (pig|cow|chicken)s?\b/` makes sense though.

Also, I don't see the immediate utility.